rocAL/rocAL_pybind/amd/rocal/readers.py

```diff
@@ -33,7 +33,8 @@ def coco(*inputs, file_root, annotations_file='', bytes_per_sample_hint=0, dump_
     #Output
     labels = []
     bboxes = []
-    kwargs_pybind = {"source_path": annotations_file, "is_output":True}
+    kwargs_pybind = {"source_path": annotations_file, "is_output":True, "mask":masks}
+    b.setSeed(seed)
```

Comment:
Please remove this setSeed instance form here

Reply:
Resolved in the latest commit